src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -50,6 +50,29 @@ def _add_host(self, host):
         completion = self.add_host(host)
         self._orchestrator_wait([completion])
         return HandleCommandResult(stdout=str(completion.result))
+        {   'cmd': "orchestrator host ls ",
+            "desc": "List hosts",
+            "perm": "rw"
+
+        }
+
+    @CLIWriteCommand("orchestrator host ls ",
+                     desc="List hosts")
+    def _list_hosts(self):
```

Comment:
FYI, 394d3d007e017d20484732a29063a6c0d5e6423a is already merged into master. You can safely remove this now.